client/src/unifyfs-internal.h

```diff
@@ -567,6 +561,7 @@ int unifyfs_set_global_file_meta(int gfid,
                                  unifyfs_file_attr_t* gfattr);
 
 int unifyfs_get_global_file_meta(int gfid,
+                                 int fid,
```

Comment:
Can we create a wrapper around unifyfs_get_global_file_meta that takes the fid?

To detangle the code, I am hoping we can keep unifyfs_set/get_global_meta clean in that they only know about gfid and a related input/output file_attr struct.

Reply:
yes, we could use a wrapper to separate the implementation concerns. we would still need to call the wrapper in all cases to make sure the local metadata gets updated.

Reply:
@adammoody I pushed a new version that creates a helper method to update local metadata from global metadata. This made more sense to me than a wrapper, since it allows us to be more precise in deciding when to do the local update.